participate_sns_swap.sh

```diff
@@ -4,8 +4,8 @@ set -euo pipefail
 
 cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
 
-export NUM_PARTICIPANTS="${1:-3}"
-export ICP_PER_PARTICIPANT="${2:-200}"
+export NUM_PARTICIPANTS="${1:-100}"
```

Comment:
Wouldn't it take too long with 100 mocked participants?

Reply:
We currently opted out for a more realistic YAML file that would resemble the tokenomics of a real-world SNS.

Currently, it takes a few minutes to participate with 100 accounts. We could reduce the number a bit, or we could parallelize the participation.

Either way (if the CI succeeds) I suggest factoring this optimization outside of this PR.